OMCompiler/SimulationRuntime/c/simulation/solver/gbode_util.c

```diff
@@ -651,7 +651,7 @@ void dumpFastStates_gb(DATA_GBODE* gbData, modelica_boolean event, double time,
  *                    -1  <= step is preliminary accepted but needs refinement
  */
 void dumpFastStates_gbf(DATA_GBODE* gbData, double time, int rejectedType) {
-  char fastStates_row[4096];
+  char fastStates_row[40960];
   unsigned int bufSize = 40960;
```

Comment:
@bernhardbachmann @AnHeuermann Based on other similar values in the file, I am guessing this is supposed to be 40960 (instead of both being 4096). Is this correct?

Reply:
It's the other way around:

```C
char fastStates_row[4096];
unsigned int bufSize = 4096;
```

We should change this into 
```
unsigned int bufSize = 4096;
char fastStates_row[bufSize];
```
to prevent this kind of typos.

Reply:
Thanks for finding the error.

Reply:
Oh my, this code is a mess.
Indeed 4096 should be large enough to print 5 numbers, but instead of checking how much memory we actually need we have just allocated `4096` bytes.

Reply:
We need to keep them as fixed size arrays not to break MSVC builds. However, we can change them to use a macro define (to be able to change it in one place.)  

I also thought 40960 was intentional because it was used in other places in the file. At least that should be lowered to 4096.

Reply:
> We need to keep them as fixed size arrays not to break MSVC builds.

Ah yes, I remembered there was a reason to don't do this, but I couldn't remember and my mwe on a Linux machine didn't complain.

Reply:
The question now is whether to keep it as 40960 or to change it to 4096? Plus, should we do it in this specific place or for all occurrences in the file? I think it is better if you take a look and decide yourself. 

40960 is not problematic by itself of course. However, if it is not needed then maybe better to fix it because it looks like a typo.

Reply:
I guess for no we can leave it and update it with the next changes in GBODE.